_posts/2023-05-31-announcing-ballerina-2201.6.0-swan-lake-update-6.md

```diff
@@ -0,0 +1,123 @@
+
+**`Bal persist` feature**
+
+One of the major highlights of Ballerina Swan Lake Update 6 is the introduction of the `Bal persist` feature. With this feature, you can now store and retrieve data from various data stores, including in-memory tables, MySQL databases, and Google Sheets. The `Bal persist` feature comprises three primary components:
+
+- **Persist model:** This allows you to define your data model by providing a structured representation of your data.
+
+    For example, consider the example model below.
+
+    ```ballerina
+    import ballerina/persist as _;
+    import ballerina/time;
+
+    type Employee record {|
+        readonly string id;
+        string firstName;
+        string lastName;
+        time:Date birthDate;
+        string gender;
+        time:Date hireDate;
+    |};
+    ``` 
+
+- **Persist CLI:** This generates a client API for your defined data model. 
+
+- **Persist client API:** This provides an interface to access and manipulate the data stored in your chosen data store. Whether you need to fetch data, update records, or perform other data-related operations, the persist client API simplifies the process.
+
+    The example code below demonstrates how CRUD operations can be performed on the persist model via the client API.       
+
+    ```ballerina
+
+    import ballerina/io;
+    import rainier.store;
+    import ballerina/persist;
+
+    store:Client sClient = check new ();
+
+    public function main() returns error? {
+        store:EmployeeInsert employee1 = {
+            id: uuid:16c6553a-373c-4b29-b1c8-c282f444248c,
+            firstName: "John",
+            lastName: "Doe",
+            gender: "Male",
+            birthDate: {
+                year: 1987,
+                month: 7,
+                day: 23
+            },
+            hireDate: {
+                year: 2020,
+                month: 10,
+                day: 10
+            }
+        };
+
+    // Create a new employee record.
+    string[] employeeIds = check sClient->/employees.post([employee1]);
+    io:println(string `Inserted employee ids: ${employeeIds.toString()}`);
+
+
+    // Get the employee record with the ID: 16c6553a-373c-4b29-b1c8-c282f444248c.
+    Employee|error employee = sClient->/employee/16c6553a-373c-4b29-b1c8-c282f444248c;
+
+
+    // Update the employee record with the ID: 16c6553a-373c-4b29-b1c8-c282f444248c.
+    Employee|error updated = sClient->/employee/16c6553a-373c-4b29-b1c8-c282f444248c.put({salary: 4000.0});
+
+
+    // Delete the employee record with the ID: 16c6553a-373c-4b29-b1c8-c282f444248c.
+    Employee|error deleted = sClient->/employee/16c6553a-373c-4b29-b1c8-c282f444248c.delete();
+
+
+    // Get all the employee records.
+    stream<Employee, error?> employee = sClient->/employee;
+    }
```

Comment:
Not formatted correctly

Reply:
Fixed via https://github.com/ballerina-platform/blog/pull/189#discussion_r1212638058.